### PR TITLE
Removed useless phpmyadmin stuff

### PR DESCRIPTION
**Pull request by ParthKolekar**
Made it raw mysql statements. 

Low chance of collisions with the engine rules now. :grin:

### CODE REVIEW

File: saathi.sql [hunk 1]
```diff
@@ -1,31 +1,3 @@
...
---
--- Database: `saathi`
---
-
--- --------------------------------------------------------
-
---
--- Table structure for table `assets`
---
-
```
Comment (gauravmittal1995) [inline saathi.sql:28]: @ParthKolekar Why are you removing the comments. What harm is it doing?? Please revert these changes.

**Comment by gauravmittal1995** [comment]
@ParthKolekar See the comments

**Comment by ParthKolekar** [comment]
@gauravmittal1995 

Let it go.
